src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,58 @@
+package duke.task;
+
+/**
+ * Todo task that inherits from Task class
+ */
+public class ToDo extends Task {
+    protected String by;
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+
+    /**
+     * Creates a new Task object and set its isDone boolean
+     *
+     * @param description details about the Task
+     * @param isDone whether Task is done or not
+     * @return Task with a corresponding description and completed status.
+     */
+    public ToDo(String description, boolean isDone) {
+        super(description, isDone);
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Checks if task contains the keyword
+     *
+     * @return whether the task contains that keyword
+     */
+    public boolean haveKeyword(String keyword) {
+        return description.contains(keyword);
+    }
+
+    /**
+     * Overrides toString method of Task class
```

Comment:
Slight note for consistency. There seem to be some javadocs with full stops while some without perhaps you can choose to settle on one?